README.md

```diff
@@ -9,6 +9,11 @@ Boss Room is a fully functional co-op multiplayer RPG made with Unity MLAPI. It
 
 Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
 
+> __IMPORTANT__:
+> - Boss Room supports those platforms supported by MLAPI (Windows and Mac). To use the WebGL platform a custom WebGL transport based on web sockets is needed.
```

Comment:
Not sure I would mention WebGL, there was zero dev on this

Reply:
that's the text for MLAPI, was there work on MLAPI on it?

Reply:
its your readme, you can remove this if you think its not valid

Reply:
Edited file